### ISSUE TITLE
Remove stylesheet colors 

### ISSUE THREAD

**Issue by marioortizmanero** [marioortizmanero/spotify-music-videos#41]
The font colors and stylesheets should be removed to support Linux themes

**Comment by marioortizmanero** [comment]
Fixed in the last commit. @AndrewAmmerlaan can you please test it? Specially the Spotify Web API part. Thanks!

**Comment by AndrewAmmerlaan** [comment]
The API selection menu looks really nice now, thanks
![Screenshot_20200102_135255](https://user-images.githubusercontent.com/8460628/71667875-2e4e8280-2d67-11ea-9c8e-033bd4058f20.png)

The Web API gives me a `Unhandled python excpetion`:
```
[13:53:17.778] INFO: Using vlc as the player
[13:53:17.779] INFO: Using SPOTIFY_WEB as the API
Traceback (most recent call last):
File "/usr/lib/python-exec/python3.7/vidify", line 11, in <module>
load_entry_point('vidify==2.0', 'console_scripts', 'vidify')()
File "/usr/lib/python3.7/site-packages/vidify/__main__.py", line 50, in main
start_gui(config)
File "/usr/lib/python3.7/site-packages/vidify/__main__.py", line 28, in start_gui
window = MainWindow(config)
File "/usr/lib/python3.7/site-packages/vidify/gui/window.py", line 77, in __init__
self.initialize_api(api_data)
File "/usr/lib/python3.7/site-packages/vidify/gui/window.py", line 108, in initialize_api
fn()
File "/usr/lib/python3.7/site-packages/vidify/gui/window.py", line 307, in init_spotify_web_api
from vidify.api.spotify.web import get_token
File "/usr/lib/python3.7/site-packages/vidify/api/spotify/web.py", line 27, in <module>
from spotipy.util import RefreshingToken, refresh_user_token
ImportError: cannot import name 'RefreshingToken' from 'spotipy.util' (/usr/lib/python3.7/site-packages/spotipy/util.py)
```

I might be using a wrong version of spotipy. I noticed that the setup.py lists spotipy-3.0 as a dependency, however the latest version I can find is 2.4.4:
```
[I] dev-python/spotipy [1]
Available versions:  (~)2.4.4 {PYTHON_TARGETS="python3_6 python3_7 python3_8"}
Installed versions:  2.4.4(15:52:20 31/12/19)(PYTHON_TARGETS="python3_7 python3_8 -python3_6")
Homepage:            https://spotipy.readthedocs.io/
Description:         A light weight Python library for the Spotify Web API

[1] "localrepo" /var/db/repos/localrepo
```

**Comment by marioortizmanero** [comment]
You can install spotipy like Travis does until it's released:

```
pip3 install git+https://github.com/felix-hilden/spotipy.git
```

**Comment by AndrewAmmerlaan** [comment]
> You can install spotipy like Travis does until it's released:

Awesome, now it works:

![Screenshot_20200102_140544](https://user-images.githubusercontent.com/8460628/71668380-f8aa9900-2d68-11ea-97e5-b64d099bc80d.png)

**Comment by marioortizmanero** [comment]
Perfect thanks for the feedback! I'm closing this issue now.

**Comment by AndrewAmmerlaan** [comment]
Hello,

Just yesterday I was working on an update to an ebuild for [QDarkStyleSheet](https://github.com/ColinDuquesnoy/QDarkStyleSheet), when I thought you might be interested in that package to. It's a dark stylesheet for PyQt5/PySide2/QtPy applications, it's the stylesheet that is used by the Spyder IDE. I haven't tested it but something as simple as this might work to implement a dark mode:
```
if use_darkstyle == True: # read `use_darkstyle` from `sys.argv` or config file
      import qdarkstyle
      app.setStyleSheet(qdarkstyle.load_stylesheet_from_environment()) # This may or may not require `QT_API` to be set
```

I can't really test it at the moment because I have an exam upcoming that I should be studying for. So I'm not sure if it works or if it looks good (screenshots on github look nice though), but it might be worth investigating because it would be a very quick and simple way to implement a dark mode.

**Comment by marioortizmanero** [comment]
Hi Andrew,

Thanks for the suggestion! I've opened up a new issue for the future (#43), QDarkStyleSheet does look like a nice solution. But I wanted to start working on Dark mode after #19 is finished. 

I'm also starting finals soon so there won't be as much activity in here until I'm done (I'll probably have time to release 2.0 when it's ready though). Best of luck in your exam!
